### ISSUE TITLE
/proc/self/fd/12 is not a regular file (use -f to see it)

### ISSUE THREAD

**Issue by subnut** [ohmyzsh/ohmyzsh#9504]
##### 1. Look for similar issues already posted (including closed ones)
None found
##### 2. Include as much relevant information as possible
On latest `omz update`, the following output was generated -
```
❯ omz update
Updating Oh My Zsh
remote: Enumerating objects: 155, done.
remote: Counting objects: 100% (143/143), done.
remote: Compressing objects: 100% (61/61), done.
remote: Total 98 (delta 68), reused 67 (delta 37), pack-reused 0
Unpacking objects: 100% (98/98), 17.19 KiB | 374.00 KiB/s, done.
From https://github.com/ohmyzsh/ohmyzsh
 * branch            master     -> FETCH_HEAD
   3f8af04..2826581  master     -> origin/master
 .github/CODEOWNERS                                     |   1 +
 lib/cli.zsh                                            |  10 +++++--
 lib/functions.zsh                                      |  19 ++-----------
 lib/git.zsh                                            |  23 ++++++++++++----
 plugins/archlinux/archlinux.plugin.zsh                 |  18 ++++++-------
 plugins/command-not-found/command-not-found.plugin.zsh |  24 +++++++++++++----
 plugins/common-aliases/README.md                       |   4 ++-
 plugins/common-aliases/common-aliases.plugin.zsh       |   2 +-
 plugins/git-prompt/gitstatus.py                        |   4 +--
 plugins/git/git.plugin.zsh                             |   1 +
 plugins/grc/README.md                                  |  37 ++++++++++++++++++++++++++
 plugins/grc/grc.plugin.zsh                             |  44 ++++++++++++++++++++++++++++++
 plugins/hanami/README.md                               |  63 ++++++++++++++++++++++++++-----------------
 plugins/hanami/hanami.plugin.zsh                       |   6 ++---
 plugins/wp-cli/README.md                               | 202 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++--------------------------------------------------------------------
 plugins/wp-cli/wp-cli.plugin.zsh                       |  51 +++++------------------------------
 tools/changelog.sh                                     |  26 +++++++++---------
 tools/check_for_upgrade.sh                             |   2 +-
 tools/upgrade.sh                                       |  10 ++++---
 19 files changed, 314 insertions(+), 233 deletions(-)
 create mode 100644 plugins/grc/README.md
 create mode 100644 plugins/grc/grc.plugin.zsh
Successfully rebased and updated refs/heads/master.
/proc/self/fd/12 is not a regular file (use -f to see it)
         __                                     __   
  ____  / /_     ____ ___  __  __   ____  _____/ /_  
 / __ \/ __ \   / __ `__ \/ / / /  /_  / / ___/ __ \ 
/ /_/ / / / /  / / / / / / /_/ /    / /_(__  ) / / / 
\____/_/ /_/  /_/ /_/ /_/\__, /    /___/____/_/ /_/  
                        /____/                       

Hooray! Oh My Zsh has been updated!
To keep up with the latest news and updates, follow us on Twitter: https://twitter.com/ohmyzsh
Want to get involved in the community? Join our Discord: https://discord.gg/ohmyzsh
Get your Oh My Zsh swag at: https://shop.planetargon.com/collections/oh-my-zsh
```
Notice the line just above the ASCII logo -
```
/proc/self/fd/12 is not a regular file (use -f to see it)
```
I have got no idea what/why it was shown.
Any ideas/explanations?

##### 3. Try to make sure the issue is due to Oh My Zsh
`omz update` is of ohmyzsh :upside_down_face:

**Comment by subnut** [comment]
In case this info is relevant...

I do use `fd` and also have it installed on my system

**Comment by mcornella** [comment]
The `fd` in question is an acronym for `file descriptor`, not the fd command. This is a bug in how the changelog output was passed to `less`, which was fixed in 81bbe86d. The next update should work correctly, but reopen the issue if it happens again.
